test_framework/scripts/longhorn-setup.sh

```diff
@@ -83,7 +83,7 @@ else
 fi
 
 
-if [[ ${LONGHORN_INFRA_TEST} ]] ; then
+if [[ ${LONGHORN_INFRA_TEST} =~ [Tt]rue ]] ; then
```

Comment:
Better add quotes for ${LONGHORN_INFRA_TEST} to ensure it's treated as a string.

Reply:
fixed